include/openssl/ssl.h

```diff
@@ -1899,6 +1899,15 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client);
 # ifndef OPENSSL_NO_CT
 
 /*
+ * A callback for verifying that the received SCTs are sufficient.
+ * Expected to return 1 if they are sufficient, otherwise 0.
+ * May return a negative integer if an error occurs.
```

Comment:
At the moment, it actually is loose in both directions (>= 1 are treated as success), which ought to be documented. However, I prefer your suggestion of erroring on anything but 1, so I'll do that in a follow-up PR.